website/src/index.js

```diff
@@ -0,0 +1,22 @@
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
+import { BrowserRouter, Switch, Route } from 'react-router-dom'
+import App from './components/App'
+import Login from './components/views/Admin/Login'
+import SignUp from './components/views/Admin/Signup'
+import Dashboard from './components/views/Admin/Dashboard'
+
+ReactDOM.render(
+
+  <BrowserRouter>
+    <Switch>
+      <Route exact path='/' render={() => <App />} />
+      <Route path='/Login' render={() => <Login />} />
+      <Route path='/Signup' render={() => <SignUp />} />
+      <Route path='/Dashboard' render={() => <Dashboard />} />
```

Comment:
This is not a good way to render components in a route because on every update that function is recreated and the whole component re-renders, while we only want that part to re-render which has been updated.
You can give the components as children of the Route to avoid this.

Reply:
Typescript was giving error with CreateBrowserHistory plugin in react

Reply:
I tried it, it didn't give me that error.
Is this what you did?

```tsx
<Route path='/Signup'>
  <SignUp />
</Route>
```

Reply:
No, I'll do that